hack/release/draft-release.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+_usage="Usage: $0 --release <RELEASE> [--latest]
+Draft the release in Github.
+
+Examples:
+  $0 --release 1.14.0 --latest      # Draft a release for v1.14.0 and force mark it as the latest release.
+  $0 --release 1.13.2               # Draft a release for v1.13.2 and mark it as the latest release based on date and version.
+
+Options:
+        --release <RELEASE>        The release to generate.
+        --all                      Mark this release as \"Latest\" (default: automatic based on date and version)."
+
+function print_usage {
+  echoerr "$_usage"
+}
+
+if ! command -v gh > /dev/null; then
+  echo "Can't find 'gh' tool in PATH, please install from https://github.com/cli/cli"
+  exit 1
+fi
+
+repo="antrea-io/antrea"
+release=""
+latest="no"
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+  --release)
+  release="$2"
+  shift 2
+  ;;
+  --latest)
+  all="yes"
```

Comment:
same typo here I think, the variable is named `latest`, not `all`

Reply:
fixed